app/models/system/network_prefix.rb

```diff
@@ -51,7 +50,7 @@ class System::NetworkPrefix < ApplicationRecord
               if: :number_max_length
             }
 
-  validates :prefix, uniqueness: { allow_blank: true }, presence: true
+  validates :prefix, uniqueness: { allow_blank: true, scope: [:number_min_length, :number_max_length] }, presence: true
```

Comment:
I think it would be great to create the custom unique validation to mark the three fields as invalid.

And in admin form when submit an invalid form, the fields number_max_length number_min_length and prefix will be red.

Reply:
It is not important currently